modules/installation-guide/partials/proc_using-the-operator-to-configure-communication-between-che-components.adoc

```diff
@@ -0,0 +1,37 @@
+[id="using-the-operator-to-configure-communication-between-{prod-id-short}-components_{context}"]
+= Using the Operator to configure communication between {prod-short} components
+
+To change between external and internal inter-component communication, update the Custom Resource.
+
+.Prerequisites
+
+* An instance of {prod-short} running on {platforms-name}.
+* You have installed {prod-short} by using the Operator.
+
+.Procedure
+
+* Configure the `__<property-value>__` value of the `disableInternalClusterSVCNames` property in the CheCluster Custom Resource server settings. Set the value of `__<property-value>__` to:
+[horizontal]
+`true`:: To use external {platforms-ingress}.
+`false`:: To use internal {orch-name} DNS names.
+
++
+====
+[source,yaml,subs="+quotes"]
+----
+apiVersion: org.eclipse.che/v1
+kind: CheCluster
+# ...
+spec:
+  server:
+    # ...
+    disableInternalClusterSVCNames: __<property-value>__
+----
+====
+
+.Verification steps
+  . Navigate to *Workloads* and select *ConfigMaps*.
```

Comment:
It works in the OpenShift UI only, we have to document using a method supported by Kubernetes: using {orch-cli}.

Reply:
More like:

```
$ {orch-cli} project {prod-namespace}
$ {orch-cli}  get configmap che -o jsonpath='{.data.CHE_KEYCLOAK_AUTH__INTERNAL__SERVER__URL}
$ {orch-cli}  get configmap che -o jsonpath='{.data.CHE_WORKSPACE_PLUGIN__REGISTRY__INTERNAL__URL}
```